src/main/java/frc/robot/subsystems/Acquisition.java

```diff
@@ -33,28 +40,28 @@ public static Acquisition getInstance() {
         return instance;
     }
 
-    /**
-     * Clamps the given speed to fit between the minimum and maximum speeds, then
-     * runs the motor at the clamped speed.
-     *
-     * @param speed The speed at which the subsystem is to run its motor.
-     */
-    public void acquire(double speed) {
-        speed = MathUtil.clamp(speed, AcquisitionConstants.minMotorSpeed, AcquisitionConstants.maxMotorSpeed);
-        driveMotor.set(speed);
+    public void acquire() {
+        intakeMotor.set(AcquisitionConstants.intakeSpeed);
     }
 
-    /**
-     * Runs the Acquistion motor backwards at a constant speed.
-     */
-    public void dispose() {
-        driveMotor.set(-AcquisitionConstants.motorSpeed);
+    public void runSushi() {
+        sushiMotor.set(AcquisitionConstants.sushiSpeed);
+    }
+
+    public void reverseMotors() {
+        sushiMotor.set(AcquisitionConstants.reverseMotorSpeed);
+        intakeMotor.set(AcquisitionConstants.reverseMotorSpeed);
     }
 
     /**
      * Stops the motor.
      */
     public void stop() {
```

Comment:
It might be useful to have 2 stop functions so that you can stop 1 motor and leave the other running